backends/google/sheets/google-sheets.js

```diff
@@ -364,6 +362,31 @@ export default class GoogleSheets extends Google {
 		return sheet?.properties?.title;
 	}
 
+	/**
+	 * Transform an array of arrays to an array of objects.
+	 * @static
+	 * @private
+	 * @param {Map<string, string>} keys A map between column indices and object keys.
+	 * @param {Array<Array<any>>} values An array of values from a spreadsheet. Each nested array corresponds to an individual spreadsheet row.
+	 * @returns {Array<Object>} An array of objects. Each object corresponds to an individual spreadsheet row.
+	 */
+	static #toObjects (keys, values) {
```

Comment:
Don't you need a corresponding `#fromObjects`?

Reply:
Right! I think I need. Will add it. Thanks!